parser/prism/Translator.h

```diff
@@ -69,7 +69,7 @@ class Translator final {
     std::unique_ptr<SorbetAssignmentNode> translateOpAssignment(pm_node_t *node);
 
     template <typename PrismLhsNode, typename SorbetLHSNode>
-    std::unique_ptr<parser::Node> translateConst(PrismLhsNode *node);
+    std::unique_ptr<parser::Node> translateConst(PrismLhsNode *node, bool skipDynamicConstantWorkaround = false);
```

Comment:
Ooooo default parameters! I forgot that C++ had those.

What exquisite taste. 🧐